action.yml

```diff
@@ -1,14 +1,11 @@
 name: "Conversation Rewards"
 description: "Compute rewards for contributors' discussion on issues that are closed as complete."
 inputs:
-  issueUrl:
-    description: "The URL the the issue needing to be parsed"
-    required: true
-  evmNetworkId:
-    description: "The EVM network ID to use"
-    required: true
-  evmPrivateEncrypted:
-    description: "The encrypted EVM private key"
+  settings:
+    description: "The configuration settings to use"
+    required: false
+  authToken:
```

Comment:
you can remove `settings` and `authToken` as they will be available in inputs object